components/VM_Arm/src/main.c

```diff
@@ -620,6 +620,29 @@ static void irq_handler(void *data, ps_irq_acknowledge_fn_t acknowledge_fn, void
     }
 }
 
+/* Default RAM initialization. Modules can overwrite this weak function with a
+ * custom initialization. e.g. to use memory shared with other components or
+ * mapped on demand.
+ */
+WEAK int init_ram_module(vm_t *vm)
+{
+    if (0 == ram_size) {
+        ZF_LOGW("VM%d: no RAM defined", vm->vm_id);
+        return 0;
+    }
+
+    ZF_LOGI("VM%d: %p - %p (RAM)", vm->vm_id, (void *)ram_base,
+            (void *)(ram_base + ram_size));
+
+    int err = vm_ram_register_at(vm, ram_base, ram_size,
+                                 vm->mem.map_one_to_one);
+    if (err) {
+        ZF_LOGE("RAM registration failed (%d)", err);
```

Comment:
For the sake of consistency, I would stick to one of "initialization", "registration" and "installing" and use that consistently.

Reply:
OK.